src/systems/filecoin_mining/sector/_index.md

```diff
@@ -11,12 +11,18 @@ entries:
 The `Sector` is a fundamental "storage container" abstraction used in Filecoin Storage Mining. It is the basic unit of storage,
 and serves to make storage conform to a set of expectations.
 
+New sectors are empty upon creation. As the miner receives client data, they fill or "pack" the piece(s) into an unsealed sector.
+
+Once a sector is full, the unsealed sector is combined by a proving tree into a single root `UnsealedSectorCID`. The sealing process then encodes an unsealed sector into a sealed sector, with the root `SealedSectorID`.
```

Comment:
Typo: `SealedSectorID` -> `SealedSector*C*ID`.

Reply:
Thanks VMX, the CBOR encoding is mentioned elsewhere in the spec as the encoding for all objects.

Reply:
@sternhenri In the published version https://filecoin-project.github.io/specs/ CBOR is mentioned here and there in sub-sections, but there is no general note that things are generally encoded as CBOR and hence also have the multicodec codec for DagCBOR.